docs/source/markdown/podman-pod-logs.1.md.in

```diff
@@ -9,15 +9,15 @@ podman\-pod\-logs - Displays logs for pod with one or more containers
 ## DESCRIPTION
 The podman pod logs command batch-retrieves whatever logs are present with all the containers of a pod. Pod logs can be filtered by container name or ID using flag **-c** or **--container** if needed.
 
-Note: Long running command of `podman pod log` with a `-f` or `--follow` needs to be reinvoked if new container is added to the pod dynamically otherwise logs of newly added containers would not be visible in log stream.
+Note: Long running command of `podman pod log` with a `-f` or `--follow` needs to be reinvoked if new container is added to the pod dynamically otherwise logs of newly added containers are not visible in log stream.
```

Comment:
```suggestion
Note: A long-running `podman pod log` command with a `-f` or `--follow` option needs to be reinvoked if a new container is added to the pod dynamically; otherwise, logs of newly added containers are not visible in the log stream.
```